RELEASE_NOTES.md

```diff
@@ -8,6 +8,7 @@ A new header is inserted each time a *tag* is created.
 ## v1.25.1
 
 - engine: add support for automatic instancing. Must be enabled with `Engine::setAutomaticInstancingEnabled(bool)`
+- engine: `Camera::getNear()` and `Camera::getCullingFar()` now return `doubles`
```

Comment:
Please update and put under `## main branch` heading

Reply:
done